.gitignore

```diff
@@ -1,3 +1,5 @@
 .vscode
 .idea
 /dist
+
+coverage.txt
```

Comment:
What's this for?

Reply:
I'm usually running unit test with the following command, which creates coverage.txt file in the current directory:

```
go test -v -short -race -timeout 30s -coverprofile=coverage.txt ./...
```

This allows checking coverage in HTML format:

![coverage txt](https://user-images.githubusercontent.com/1322923/133378189-5648bab7-3436-4425-9270-576e580dd92d.png)